integration_tests/infinite_bus.feature

```diff
@@ -0,0 +1,13 @@
+Feature: Infinite Bus tests
```

Comment:
Can you add one more integration_test to InfiniteBusStrategy for balancing_market?

Reply:
Can you please specify what needs to be tested?

Reply:
Great, then there is not need for an integration tests, IMO.